pyface/__init__.py

```diff
@@ -54,7 +54,7 @@ def load_tests(loader, standard_tests, pattern):
     """
     from os import environ
     from os.path import dirname
-    from pyface.util.testing import filter_tests, is_traits_version_ge
```

Comment:
Technically - yes, this is changing the public API but it would have been a mistake to do `from pyface import is_traits_version_ge` or anything of that sort so im okay with making this change.